docs/release-notes/RELEASE-NOTES-0.9.0.md

```diff
@@ -6,45 +6,51 @@ It:
 * Does that TODOFILLMEIN. This impacts the persisted DB;
 * Does that TODOFILLMEIN.
 
-This release introduces backward incompatible changes in the chain format.
-After upgrading your node, you will not have your previous balance (even if you keep your key pair).
+This release introduces backward incompatible changes in the chain format:
+* After upgrading your node, you will not have your previous balance (even if you keep your key pair);
+* Please ensure that you do not reuse a persisted blockchain produced by the previous releases "v0.8.x".
 
 Please join the testnet by following the instructions below, and let us know if you have any problems by [opening a ticket](https://github.com/aeternity/epoch/issues).
 Troubleshooting of common issues is documented [in the wiki](https://github.com/aeternity/epoch/wiki/Troubleshooting).
 
 The instructions below describe:
 * [How to retrieve the released software for running a node](#retrieve-the-software-for-running-a-node);
+* [How to install a node](#install-node);
 * [How to join the testnet](#join-the-testnet).
 
 ## Retrieve the software for running a node
 
-Download the [release binary](https://github.com/aeternity/epoch/releases/tag/v0.9.0) corresponding to your platform, e.g. `epoch-0.9.0-osx-10.12.6.tar.gz`; you would normally find the downloaded package in `~/Downloads` on macOS.
+You can run a node by using either:
+* The published [release binary](https://github.com/aeternity/epoch/releases/tag/v0.9.0) corresponding to your platform; or
+* The published [Docker image `aetrnty/epoch`](https://github.com/aeternity/epoch/blob/v0.9.0/docs/docker.md).
 
-The binaries are tested on the following platforms:
+The release binaries are tested on the following platforms:
 * Ubuntu 16.04.3 LTS (x86-64);
 * macOS Sierra (x86-64);
 * macOS High Sierra (x86-64).
 
-The macOS package has a hard dependency on OpenSSL v1.0.0 installed with [Homebrew](https://brew.sh/) in its default path `/usr/local/opt/openssl/lib/libcrypto.1.0.0.dylib` and also a hard
-dependency on libsodium v1.0.16 installed with [Homebrew](https://brew.sh/) also in its default path `/usr/local/opt/libsodium/lib/libsodium.23.dylib`.
-In case you have installed either of them in a non-default path, you could use symlink(s) to work around the issue.
+The user configuration is documented in the [wiki](https://github.com/aeternity/epoch/wiki/User-provided-configuration).
```

Comment:
IMO this line belongs to configuration.md

Reply:
I considered that, though I would prefer not to perform this change in this iteration.

I feel there is distinction between how node *can* be configured, and how node is *meant* to be configured. I kept this line for the former, and I put the latter in configuration.md, whose scope I describe as "how to configure your epoch node installed using a release binary for joining a public network of nodes (e.g. testnet) knowing an initial network peer to join."